### PR TITLE
Bug fixes for mongodb compatibility and length of description fields

### PR DESCRIPTION
**Pull request by rjzondervan**
This contains 4 small fixes:
- Character limits are unset for description fields in SQL databases
- MongoDB Creation of metadata is fixed to prevent duplicates
- MongoDB creation of organisations is fixed
- Idem for themes

### CODE REVIEW

File: lib/Controller/ThemesController.php [hunk 1]
```diff
@@ -110,11 +110,16 @@ public function show(string $id, ObjectService $objectService): JSONResponse
                 'mongodbCluster' => $this->config->getValueString($this->appName, 'mongodbCluster')
             ];
 
-            $filters['_id'] = (string) $id;
 
-            $result = $objectService->findObject($filters, $dbConfig);
 
-            return new JSONResponse($result);
+			$data['_schema'] = 'organisation';
```
Comment (bbrands02) [inline lib/Controller/ThemesController.php:115]: Moet het _schema hier 'theme' zijn?

**Review by bbrands02 — CHANGES_REQUESTED** [review]

File: lib/Migration/Version6Date20240923093806.php [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+		 * @var ISchemaWrapper $schema
+		 */
+		$schema = $schemaClosure();
+
+		if($schema->hasTable(tableName: 'publications') === true) {
+			$table = $schema->getTable(tableName: 'publications');
+			if($table->hasColumn(name: 'description') === true) {
+				$column = $table->getColumn(name: 'description');
+
+				var_dump('unsetting max length');
```
Comment (WilcoLouwerse) [inline lib/Migration/Version6Date20240923093806.php:47]: var dump

**Review by WilcoLouwerse — CHANGES_REQUESTED** [review]
probs better to remove all var_dumps? 
Optionally you could add a description to the migration, if this is possible here just like we can do for our common-gateway repo

File: lib/Controller/ThemesController.php (hunk 1 above)
Comment (rjzondervan) [reply lib/Controller/ThemesController.php:115]: Scherp, pas ik aan

File: lib/Migration/Version6Date20240923093806.php (hunk 2 above)
Comment (rjzondervan) [reply lib/Migration/Version6Date20240923093806.php:47]: Will remove

**Review by WilcoLouwerse — APPROVED** [review]

**Review by WilcoLouwerse — APPROVED** [review]

**Review by bbrands02 — APPROVED** [review]

**Pull request merged**